docker-compose.yml

```diff
@@ -42,6 +42,7 @@ services:
       - PYTHONWARNINGS=${PYTHONWARNINGS:-default}
       - MAINTENANCE_MODE=${MAINTENANCE_MODE:-False}
       - REVISION_HASH=${KUMA_REVISION_HASH:-undefined}
+      - SENTRY_DSN=${SENTRY_DSN:-}
```

Comment:
This is only there for local development so that the environment variable becomes available inside the container.